script/Ciel/Day 13/13_01_02_CIEL13_5_2.txt

```diff
@@ -355,18 +402,26 @@ Even though she was still alive――even though she was still human――they t
 }
 [sha:b4ab74cd94711e050298f7943588795554cd89bb]{
 -- Page 89, Offset 35515.
--- それだけの贖罪をして、まだ―――
-But even after going through all that suffering so she could atone for her sins, she still can't―――
+-- それだけの贖罪をして、まだ
+But despite all that penitence,
+ all that pain, she still can't―――
 }
 [sha:1113d2e2989c00d0c6412b3add513c0b9c102540]{
 -- Page 90, Offset 35516.
 -- “でも、そんなのは違うんです。私はぜんぜん強くない”
-"But you're wrong. I'm not strong at all."
+// Referencing 28022.
+// Voiced
+"But, that isn't even close to being true.
+ I'm not strong at all."
 }
 [sha:7b6df9fffcdd53e0506814f50acfa917021ea497]{
 -- Page 90, Offset 35517.
 -- “単に死なない体だから、体が強いから罰に耐えられただけ。普通の体だったなら、きっと一日で投げ出していたんです”
-"I was only able to withstand it because I have a sturdy body that refuses to die. If I had a normal body, I'm sure I would've given up on the first day."
+// Referencing 28023.
+// Voiced
+"I simply have a body that cannot die,
+ and so my %{i}body%{/i} was strong enough to withstand that punishment.
+ If I'd had a regular body, I'd have thrown in the towel after only a day."
 }
 [sha:c1efeea9ec40664e87b0a307385510bfd0028fbd]{
 -- Page 90, Offset 35518.
```

Comment:
35519
ellipsis. possibly could use a more harsh wording 
also the pause, maybe

>Don't... don't give me that shit!